test_scripts/API/RAI/017_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_4_Apps_RAI_steeringWheelLocation.lua

```diff
@@ -0,0 +1,262 @@
+          text_field("menuTitle"),
+          text_field("locationName"),
+          text_field("locationDescription"),
+          text_field("addressLines"),
+          text_field("phoneNumber")
+        },
+        imageFields =
+        {
+          image_field("softButtonImage"),
+          image_field("choiceImage"),
+          image_field("choiceSecondaryImage"),
+          image_field("vrHelpItem"),
+          image_field("turnIcon"),
+          image_field("menuIcon"),
+          image_field("cmdIcon"),
+          image_field("showConstantTBTIcon"),
+          image_field("locationImage")
+        },
+        mediaClockFormats =
+        {
+          "CLOCK1",
+          "CLOCK2",
+          "CLOCK3",
+          "CLOCKTEXT1",
+          "CLOCKTEXT2",
+          "CLOCKTEXT3",
+          "CLOCKTEXT4"
+        },
+        graphicSupported = true,
+        imageCapabilities = { "DYNAMIC", "STATIC" },
+        templatesAvailable = { "TEMPLATE" },
+        screenParams =
+        {
+          resolution = { resolutionWidth = 800, resolutionHeight = 480 },
+          touchEventAvailable =
+          {
+            pressAvailable = true,
+            multiTouchAvailable = true,
+            doublePressAvailable = false
+          }
+        },
+        numCustomPresetsAvailable = 10
+      },
+      audioPassThruCapabilities =
+      {
+        samplingRate = "44KHZ",
+        bitsPerSample = "8_BIT",
+        audioType = "PCM"
+      },
+      hmiZoneCapabilities = "FRONT",
+      softButtonCapabilities =
+      {
+        shortPressAvailable = true,
+        longPressAvailable = true,
+        upDownAvailable = true,
+        imageSupported = true
+      }
+    })
+	end)
+end
+
+function Test:Precondition_connectMobile()
+	self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+	self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession:StartService(7)
+end
+
+function Test:Precondition_SecondSession()
+	self.mobileSession1 = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession1:StartService(7)
+end
+
+function Test:Precondition_ThirdSession()
+	self.mobileSession2 = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession2:StartService(7)
+end
+
+function Test:Precondition_FourthSession()
+	self.mobileSession3 = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession3:StartService(7)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_RAI_FirstApp_steeringWheelLocation()
+	local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success=true, resultCode = "SUCCESS", steeringWheelLocation = "CENTER" })
```

Comment:
@istoimenova 
As I understood parameter steeringWheelLocation should be part of hmiCapabilities.
The same issue is appeared in next two test steps
Could you, please, check?